fossunited/fossunited/doctype/foss_event_schedule/foss_event_schedule.py

```diff
@@ -5,4 +5,25 @@
 
 
 class FOSSEventSchedule(Document):
```

Comment:
Nope, not a new doctype, this doctype is already there in the desk and was recently used for creating schedules for [PuneFOSS 2.1](https://fossunited.org/events/punefoss?tab=schedule)

Since it is an existing doctype, lets rename it later with all the other doctypes.

Reply:
resolving the comment as this isnt a new doctype.